tests/drivers/test_RS_SGS100A.py

```diff
@@ -0,0 +1,80 @@
+from collections.abc import Sequence
+from typing import Any
+
+import numpy as np
+import pytest
+
+from qcodes.instrument_drivers.rohde_schwarz.SGS100A import RohdeSchwarz_SGS100A
+
+
+@pytest.fixture(scope='function')
+def sg():
```

Comment:
```suggestion
@pytest.fixture(scope='function', name="sg")
def _make_sg():
```

If you follow this pattern you can avoid the pylint warning about name shadowing name from outer scope

Reply:
I was wondering how to address that.  Thanks for the suggestion!